packages/0x.js/test/utils/token_utils.ts

```diff
@@ -3,7 +3,7 @@ import * as _ from 'lodash';
 import {InternalZeroExError, Token} from '../../src/types';
 
 const PROTOCOL_TOKEN_SYMBOL = 'ZRX';
-const WETH_TOKEN_SYMBOL = 'WETH';
+const WETH_TOKEN_SYMBOL = 'WETH9';
```

Comment:
According to the Solidity source code, the symbol should still be `WETH`... What is going on here?